File: 프로그래머스/1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.java
class Solution {
    public String solution(String phone_number) {
        String answer = "";
        String number = "";
        for(int i=phone_number.length()-4; i<phone_number.length(); i++){
            number+=phone_number.charAt(i);
        }
        for(int i=0; i<phone_number.length()-4; i++){
            answer+='*';
        }
        answer=answer+number;
        return answer;
    }
}

File: 프로그래머스/1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.js
function solution(phone_number) {
        let answer = "";
        let number = "";
        for(let i=phone_number.length-4; i<phone_number.length; i++){
            number+=phone_number.charAt(i);
        }
        for(let i=0; i<phone_number.length-4; i++){
            answer+='*';
        }
        answer=answer+number;
    return answer;
}